uniffi_meta/src/types.rs

```diff
@@ -57,6 +57,7 @@ pub enum ExternalKind {
     Trait,
     // Either a record or enum
     DataClass,
+    Error,
```

Comment:
I don't think this is going to work - because errors can be enums of interfaces, I'd think that just `ExternalKind::DataClass` and `ExternalKind::Interfaces` would be OK. Why did you find it necessary to add this new discriminant?

Reply:
I needed a way to differentiate errors from other types in order to provide the appropriate forward macro, https://github.com/mozilla/uniffi-rs/pull/2265/files#diff-6cbadee39d35e495122970278e12fc509f5243d4e3279a911112f69e798fe247R11-R12.

Reply:
It's possible you might be able to use `ci.is_name_used_as_error()`, eg, https://github.com/mozilla/uniffi-rs/blob/1c8dd50de3413c03b7526aa9cab739049b64d331/uniffi_bindgen/src/scaffolding/templates/scaffolding_template.rs#L19

Reply:
👍 , that seems to work. I pushed an update to this effect.